android/engine/src/main/java/org/smartregister/fhircore/engine/ui/questionnaire/QuestionnaireViewModel.kt

```diff
@@ -184,12 +185,15 @@ constructor(
         bundle.entry.forEach { bun ->
           // add organization to entities representing individuals in registration questionnaire
           if (bun.resource.resourceType.isIn(ResourceType.Patient, ResourceType.Group)) {
-            appendOrganizationInfo(bun.resource)
+            if (questionnaireConfig.setOrganizationDetails) {
```

Comment:
Shouldn't we be adding these details to any resource that has org/practitioner properties?

Reply:
How we would be able to know in advance which resource has org/practitioner properties. should I apply checks to make sure particular resource has org/practitioner properties. Correct me if I misunderstood this question.

Reply:
we can have this for Patient/Group resource for now. But the idea is to be able to set the details in any FHIR resource that supports it.